File: ueueo-framework/ueueo-uow/src/main/java/com/ueueo/uow/UnitOfWorkEventArgs.cs
using System;
using JetBrains.Annotations;

namespace Volo.Abp.Uow;

public class UnitOfWorkEventArgs : EventArgs
{
    /**
     * Reference to the unit of work related to this event.
    */
    public IUnitOfWork UnitOfWork;//  { get; }

    public UnitOfWorkEventArgs(@Nonnull IUnitOfWork unitOfWork)
    {
        Objects.requireNonNull(unitOfWork, nameof(unitOfWork));

        UnitOfWork = unitOfWork;
    }
}

File: ueueo-framework/ueueo-uow/src/main/java/com/ueueo/uow/UnitOfWorkEventArgs.java
package com.ueueo.uow;

import lombok.Data;
import org.springframework.lang.NonNull;

import java.util.Objects;

@Data
public class UnitOfWorkEventArgs extends EventArgs {
    /**
     * Reference to the unit of work related to this event.
     */
    private IUnitOfWork unitOfWork;

    public UnitOfWorkEventArgs(@NonNull IUnitOfWork unitOfWork) {
        Objects.requireNonNull(unitOfWork);
        this.unitOfWork = unitOfWork;
    }
}
